events/2016/london.html

```diff
@@ -76,7 +76,7 @@ <h3 class="center-text">Apply Now!</h3>
       <div class="container">
         <h2 class="center-text">Join us!</h2>
         <p>The ideal student for a NodeTogether event is a new or junior developer who is interested in getting started with Node.js. We don’t expect you to have any programming experience and will be spending a lot of time getting a development environment set up.</p>
-       <p>Additionally, we are particularly interested in people from underrepresented groups, including but not limited to, women, people of color, people with physical or mental handicaps, and LGBTQ  people.</p>
+       <p>Additionally, we are particularly interested in people from underrepresented groups, including but not limited to, women, people of color, people with mental and/or physical disabilities, and LGBTQ and non-binary people.</p>
```

Comment:
You have two `and` in here: , `and LGBTQ and non-binary`. This would work better gramatically if it was changed to `, , LGBTQ, and non-binary`.